### PR TITLE
CodeBase Fix : Teams Page

### PR DESCRIPTION
**Pull request by imabp**
#184

### CODE REVIEW

File: src/App.js [hunk 1]
```diff
@@ -4,7 +4,7 @@ import Hidden from '@material-ui/core/Hidden';
 import "./index.css";
 // importing pages 
 import Index from './pages/home/index'
-import Team from './pages/team'
+import Team from './pages/team/index'
```
Comment (Souvikns) [inline src/App.js:7]: if there is an `index.js` file inside a folder then you can use default import from there and not add the the `index.js` file in the import path. 
```js
import Team from './pages/team'
```
would do just remove the previous files.

File: src/pages/team/index.jsx [hunk 2]
```diff
@@ -1,13 +1,23 @@
...
-        </div>
+const Team = () => {
+
+    const data = team.useTeam()
+    console.log(data)
+    return (
+        <>
+            <OurStory/>
+       
+            <DisplayTeams members={data} />
```
Comment (Souvikns) [inline src/pages/team/index.jsx:19]: As you are sending data inside the `displayteams` component create `PropTypes` for it.

**Review by Souvikns — CHANGES_REQUESTED** [review]

File: src/pages/team/index.jsx (hunk 2 above)
Comment (imabp) [reply src/pages/team/index.jsx:19]: yes will create after next PR

Comment (imabp) [reply src/pages/team/index.jsx:19]: This is just file structure fixes

File: src/App.js (hunk 1 above)
Comment (imabp) [reply src/App.js:7]: Ohk let me do it in next pr

**Pull request merged**